src/core/lib/security/credentials/tls/grpc_tls_certificate_verifier.cc

```diff
@@ -30,6 +30,15 @@
 
 namespace grpc_core {
 
+namespace {
+const char kTlsCertificateVerifierTypeExternal[] = "external";
```

Comment:
Suggest making this a private static member of the `ExternalCertificateVerifier` class, so that it's not possible for an external impl to accidentally use it.